src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -650,13 +670,19 @@ void Convolution::setPostOps(dnnl::primitive_attr& attr,
             }
 
             if (useLegacyPostOps) {
+                bool forceBinary = false;
+                // Temporary debug functionality to be able to force binary postops for any model
+                CPU_DEBUG_CAP_ENABLE(if (std::getenv("OV_CPU_FORCE_FQ_AS_BINARY")) {forceBinary = true;})
```

Comment:
Do we still need this?

Reply:
I would propose to keep it for debugging for now.
This is only temporarily, I am pretty sure after migration is done till the release we need to have a flexible way to tweak those heuristics.
Can be removed before release of course.

Reply:
Considering this is hided under Debug Caps I am ok to leave as is. However we have to either delete it once migration period is over or properly document inside corresponding developer doc page

Reply:
Yes, if we want to keep it even for release, than we should implement it in proper way, like any other debug caps environment option.